### PR TITLE
Fix for seahorse 

### PR DESCRIPTION
**Pull request by PPAC37**
as explaine in https://github.com/MarginallyClever/Makelangelo-software/issues/353#issuecomment-1039446478
avoid having some drawn lines which should be displacements.

### CODE REVIEW

File: src/main/java/com/marginallyclever/makelangelo/makeArt/io/vector/LoadSVG.java [hunk 1]
```diff
@@ -367,7 +367,7 @@ private void doCubicCurveAbs(SVGPathSeg item, Matrix3d m) {
 		ArrayList<Point2D> points = b.generateCurvePoints(0.1);
```
Comment (coliss86) [inline src/main/java/com/marginallyclever/makelangelo/makeArt/io/vector/LoadSVG.java:367]:
```suggestion
		List<Point2D> points = b.generateCurvePoints(0.1);
```

Comment (i-make-robots) [reply src/main/java/com/marginallyclever/makelangelo/makeArt/io/vector/LoadSVG.java]: why is it better to use generic List?  One extra include....

Comment (coliss86) [reply src/main/java/com/marginallyclever/makelangelo/makeArt/io/vector/LoadSVG.java]: you don't need the method specific to an ArrayList in this part of the code, you just need a List which is the interface.

Comment (i-make-robots) [reply src/main/java/com/marginallyclever/makelangelo/makeArt/io/vector/LoadSVG.java]: I don't need an extra include, either!  If I had a factory that was returning a List then sure, generic allows for more flex.  But here it's of no benefit.

Comment (coliss86) [reply src/main/java/com/marginallyclever/makelangelo/makeArt/io/vector/LoadSVG.java]: `import` are generated by the IDE, so it is not an issue. your main problem is why ArrayList over List. I may not using the right word to explain so, instead, I suggest you to read this small answer: https://stackoverflow.com/questions/147468/why-should-the-interface-for-a-java-class-be-preferred

**Pull request merged**